Packs/IronPort/Integrations/CiscoEmailSecurityApplianceIronPortV2/README.md

```diff
@@ -1,5 +1,5 @@
 The Cisco Email Security Appliance is an email security gateway product. It is designed to detect and block a wide variety of email-born threats, such as malware, spam and phishing attempts.
```

Comment:
Please regenerate this README file from the YAML file with updated descriptions.